hyperspy/misc/eds/utils.py

```diff
@@ -243,6 +247,72 @@ def take_off_angle(tilt_stage,
                                   + math.sin(a) * math.sin(c)))
 
 
+def xray_lines_model(elements=['Al', 'Zn'],
```

Comment:
```python
if elements is None:
raise Warning (no elements specified - please specify some)
```
I would rather just not have a default value for `elements`, i.e. make it non-optional. As this is a utility function where you never gain access to the intermediate model, calling it without `elements` just doesn't make sense...